docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+services:
+  db:
+    image: postgres:17.2-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5433:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command to run the Django application includes `wait_for_db`, `migrate`, and `runserver`. Consider using a production-ready server like Gunicorn for deployment instead of Django's development server (`runserver`).